.github/workflows/Tests.yaml

```diff
@@ -602,6 +602,158 @@ ${{ env.CACHE_KEY_SUFFIX }}"
+      - name: Configure Spack
+        # - To avoid re-building packages that are already installed by Homebrew
+        #   we let Spack find them.
+        # - Add the dependency cache as binary mirror.
+        run: |
+          source $HOME/spack/share/spack/setup-env.sh
+          spack debug report
+          spack compiler find && spack compiler list
+          spack external find && spack config get packages
+          spack mirror add dependencies file://$HOME/dependencies/spack
+      # Install the remaining dependencies from source with Spack. We install
+      # them in an environment that we can activate later. After building the
+      # dependencies from source we cache them as compressed tarballs.
+      - name: Install Spack dependencies
+        run: |
+          source $HOME/spack/share/spack/setup-env.sh
+          spack env create spectre support/DevEnvironments/spack.yaml
+          spack env activate spectre
+          spack remove jemalloc
+          spack concretize --reuse
+          spack install --no-check-signature
+          spack find -v
+      # Replace the ccache directory that building the dependencies may have
+      # generated with the cached ccache directory.
+      - name: Clear ccache from dependencies
+        run: |
+          ccache --clear
+          rm -rf $CCACHE_DIR
+          mkdir -p $CCACHE_DIR
+      - name: Restore ccache
+        uses: actions/cache@v2
+        with:
+          path: ~/ccache
+          key: "ccache-macos-${{ github.ref }}-${{ env.CACHE_KEY_SUFFIX }}"
+          restore-keys: |
+            ccache-macos-refs/heads/develop-
+      - name: Configure ccache
+        run: |
+          ccache -pz
+      # Configure, build and run tests. See the `unit_tests` job above for
+      # details.
+      # - We increase the timeout for tests because the GitHub-hosted macOS VMs
+      #   appear to be quite slow.
+      - name: Configure build with cmake
+        run: |
+          source $HOME/spack/share/spack/setup-env.sh
+          spack env activate spectre
+          mkdir build && cd build
+          cmake \
+            -D CMAKE_C_COMPILER=clang \
+            -D CMAKE_CXX_COMPILER=clang++ \
+            -D CMAKE_Fortran_COMPILER=gfortran-11 \
+            -D CMAKE_CXX_FLAGS="-Werror" \
+            -D OVERRIDE_ARCH=x86-64 \
+            -D BUILD_SHARED_LIBS=ON \
+            -D BUILD_PYTHON_BINDINGS=ON \
+            -D MEMORY_ALLOCATOR=SYSTEM \
+            -D CHARM_ROOT=$(spack location --install-dir charmpp) \
+            -D CMAKE_BUILD_TYPE=Debug \
+            -D DEBUG_SYMBOLS=OFF \
+            -D UNIT_TESTS_IN_TEST_EXECUTABLES=OFF \
+            -D STUB_EXECUTABLE_OBJECT_FILES=ON \
+            -D STUB_LIBRARY_OBJECT_FILES=ON \
+            -D USE_PCH=ON \
+            -D USE_CCACHE=ON \
+            -D SPECTRE_TEST_TIMEOUT_FACTOR=5 \
+            $GITHUB_WORKSPACE
+      - name: Build tests
+        working-directory: build
+        run: |
+          make -j4 RunTests
```

Comment:
I believe that only two are available so `-j2`

Reply:
squashed this in and rebased